### ISSUE TITLE
mariadb: binary logs retention

### ISSUE THREAD

**Issue by dwlfrth** [inverse-inc/packetfence#2440]
MariaDB binary logs can fill up disk space quite fast when used in an active-active with DB replication environment.

- `expire_logs_days` should be explicitly set (default is 0)
- `max_binlog_size` should be explicitly set (default is 1073741824)

**Comment by dwlfrth** [comment]
After discussion with @louismunro, the following should fit our needs

- `expire_logs_days` -> 2
- `max_binlog_size` -> 13107200

**Comment by julsemaan** [comment]
In order to fix this on 7.0 and 7.1, we need to add the following to the bottom of /usr/local/pf/conf/mariadb/mariadb.conf.tt on each server:
```
expire_logs_days=2
max_binlog_size=13107200
```

And then restart packetfence-mariadb (on each server again):
```
# service packetfence-mariadb restart
```
